src/KubernetesClient/Util/Common/IRunnable.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace k8s.Util.Common
+{
+    public interface IRunnable : IDisposable
```

Comment:
runnable? is this java's runnable?

Reply:
It was inspired by the java runnable, yes.

Reply:
why not task.run(() => {})
i remember `irunnable` is back to java6? not replacing by lambda?